master/buildbot/process/remotecommand.py

```diff
@@ -51,61 +59,75 @@ class RemoteCommand(base.RemoteCommandImpl):
 
     def __init__(
         self,
-        remote_command,
-        args,
-        ignore_updates=False,
-        collectStdout=False,
-        collectStderr=False,
-        decodeRC=None,
-        stdioLogName='stdio',
-    ):
+        remote_command: str,
+        args: dict[str, Any],
+        ignore_updates: bool = False,
+        collectStdout: bool = False,
+        collectStderr: bool = False,
+        decodeRC: dict[int, int] | None = None,
+        stdioLogName: str = 'stdio',
+    ) -> None:
         if decodeRC is None:
             decodeRC = {0: SUCCESS}
-        self.logs = {}
-        self.delayedLogs = {}
-        self._closeWhenFinished = {}
-        self.collectStdout = collectStdout
-        self.collectStderr = collectStderr
+        self.logs: dict[str, StreamLog] = {}
+        self.delayedLogs: dict[
+            str,
+            tuple[
+                Callable[[RemoteCommand], Awaitable[StreamLog]],
+                bool,
+            ],
+        ] = {}
+        self._closeWhenFinished: dict[str, bool] = {}
+        self.collectStdout: bool = collectStdout
+        self.collectStderr: bool = collectStderr
         self.stdout = ''
         self.stderr = ''
-        self.updates = {}
-        self.stdioLogName = stdioLogName
-        self._startTime = None
-        self._remoteElapsed = None
+        self.updates: defaultdict[str, list[Any]] = defaultdict(list)
```

Comment:
This is not just type change, best would be to split it into separate commit. Same with line boundary finders.

Reply:
Indeed, missed it, sorry about that.